frontend/src/components/DetailsPanel/index.js

```diff
@@ -131,6 +133,34 @@ function DetailsPanel({ classId, classList, setClassId, children, dynamicShiftbu
     }
   };
 
+  const renderInstructorInfo = () => {
+    if (!shiftDetails || !shiftDetails.shift_type || !panelInfo?.instructor_email) return;
+  
+    return (
+      <>
+        {panelInfo?.instructor_f_name && panelInfo?.instructor_l_name
+          ? `${panelInfo.instructor_f_name} ${panelInfo.instructor_l_name}`
+          : "No instructor available"}
+        {shiftDetails.shift_type === "my-shifts" 
```

Comment:
Thinking about the logic again, I think we should also display email if it is the user's shift that they have requested coverage for (since they can't make the shift). These are the shifts that are type "my-coverage-requests"